### PR TITLE
code to make truth_summary tables for stars

### PR DESCRIPTION
**Pull request by jchiang87**
I'll add the variability truth table code in a separate PR.

### CODE REVIEW

**Comment by jchiang87** [comment]
I think this is ready to merge.

File: python/desc/sims_truthcatalog/star_truth.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            Limit on number of rows to return from the query to the star
+            database.  If None, then no limit will be applied.
+        """
+        self.outfile = outfile
+        if os.path.isfile(outfile):
+            raise OSError(f'{outfile} already exists.')
+        if not os.path.isfile(star_db_file):
+            raise FileNotFoundError(f'{star_db_file} not found.')
+        self.conn = sqlite3.connect(star_db_file)
+        query = 'select * from stars'
```
Comment (JoanneBogart) [inline python/desc/sims_truthcatalog/star_truth.py]: Better to select needed columns explicitly

**Review by JoanneBogart — COMMENTED** [review]
I assume a command-line script to write star truth will be added to bin.src at some point; it doesn't have to be part of this merge.  That's the only comment I have other than the one on use of 'select *'

File: python/desc/sims_truthcatalog/star_truth.py (hunk 1 above)
Comment (JoanneBogart) [reply python/desc/sims_truthcatalog/star_truth.py]: Looks fine - ready to merge as far as I'm concerned.

**Pull request merged**